tests/functional/spec/components/dropdown.spec.js

```diff
@@ -0,0 +1,138 @@
+            .click(DropdownSummaryPage.dropdownMandatoryAnswerEdit())
+            .click(DropdownMandatoryPage.submit())
+            .getText(DropdownSummaryPage.dropdownMandatoryAnswer()).should.eventually.contain("Rugby is better!")
+            .click(DropdownSummaryPage.dropdownMandatoryAnswerEdit())
+            .selectByValue(DropdownMandatoryPage.answer(), "Liverpool")
+            .click(DropdownMandatoryPage.submit())
+            .getText(DropdownSummaryPage.dropdownMandatoryAnswer()).should.eventually.contain("Liverpool");
+        });
+    });
+  });
+
+  describe('Given I start a Mandatory With Overridden Error Dropdown survey', function() {
+
+    const DropdownMandatoryPage = require('../../pages/components/dropdown/mandatory_with_overridden_error/dropdown-mandatory-with-overridden-error.page');
+
+    before(function() {
+      return helpers.openQuestionnaire('test_dropdown_mandatory_with_overridden_error.json');
+    });
+
+    it('When I have not selected a dropdown option and click Continue, Then the overridden error message should be displayed', function() {
+      return browser
+        .click(DropdownMandatoryPage.submit())
+        .getText(DropdownMandatoryPage.errorNumber(1)).should.eventually.contain("Overridden test error message.");
+    });
+  });
+
+  //Optional
+  describe('Given I start a Optional Dropdown survey', function() {
+
+    const DropdownOptionalPage = require('../../pages/components/dropdown/optional/dropdown-optional.page');
+    const DropdownSummaryPage = require('../../pages/components/dropdown/optional/summary.page');
+    const schema = 'test_dropdown_optional.json';
+
+    it('When I have not selected a dropdown option, Then the summary should display "No answer provided"', function() {
+      return helpers.openQuestionnaire(schema)
+        .then(() => {
+          return browser
+            .click(DropdownOptionalPage.submit())
+            .getText(DropdownSummaryPage.dropdownOptionalAnswer()).should.eventually.contain("No answer provided");
+        });
+    });
+
+    it('When I have selected a dropdown option, Then the selected option should be displayed in the summary', function() {
+      return helpers.openQuestionnaire(schema)
+        .then(() => {
+          return browser
+            .selectByValue(DropdownOptionalPage.answer(), "Rugby is better!")
+            .click(DropdownOptionalPage.submit())
+            .getText(DropdownSummaryPage.dropdownOptionalAnswer()).should.eventually.contain("Rugby is better!");
+        });
+    });
+
+    it('When I have selected a dropdown option and I reselect the default option (Please select an answer etc), Then the summary should display "No answer provided"', function() {
+      return helpers.openQuestionnaire(schema)
+        .then(() => {
+          return browser
+            .selectByValue(DropdownOptionalPage.answer(), "Chelsea")
+            .selectByValue(DropdownOptionalPage.answer(), "")
```

Comment:
Not sure of the value of changing the option without posting. I think we should be submitting the form, then going back to change to no answer.